hw-probe.pl

```diff
@@ -3753,6 +3753,18 @@ ()
         }
     }
     
+    if(not $Sys{"Type"} and not $Sys{"Model"})
+    {
+        for my $str ($Kldstat_v) {
```

Comment:
May be rewrite this like the following?

    if($Kldstat_v=~/bcm2835_/)
    {
        $Sys{"Type"} = "system on chip";
        $Sys{"Model"} = "Raspberry Pi";
    }

Reply:
yes, was not touching perl a long time :)

```
System: freebsd-13.0-BETA4
Arch:   arm64
Kernel: 13.0-BETA4
Model:  Raspberry Pi
Type:   system on chip
Hwaddr: 0fed5e133edb751c1b8e0b2f86740ab3
```

Reply:
Please update your commit accordingly.

Reply:
Done )